docusaurus/docs/migrating-from-toolkit.md

```diff
@@ -3,19 +3,19 @@ id: migrating-from-toolkit
 title: Migrating from Grafana Toolkit
 ---
 
-If you were using `@grafana/toolkit` to scaffold and develop plugins this guide should help you make the jump to plugin tools.
+If you are using `@grafana/toolkit` to scaffold and develop plugins this guide will help you make the jump to plugin tools.
```

Comment:
is this the place we wanted to talk about "why migrate from toolkit"? https://grafana.slack.com/archives/C3HJV5PNE/p1676508337414769?thread_ts=1675440138.901209&cid=C3HJV5PNE

Reply:
Thank you for addressing my questions. Would be great to understand the timeline and benefits of migrating.